python/dolfinx/fem/forms.py

```diff
@@ -19,9 +19,8 @@
 
 
 class Form:
-    def __init__(self, form: ufl.Form, dtype: np.dtype = PETSc.ScalarType,
-                 form_compiler_parameters: dict = {}, jit_parameters: dict = {}):
-        """Create DOLFINx Form
```

Comment:
Why remove typing on parameters?

Reply:
Can also take collrction of forms

Reply:
Fixed (partially, some types depend on the metaclass type).